discovery/v1.ts

```diff
@@ -170,7 +177,7 @@ class DiscoveryV1 extends GeneratedDiscoveryV1 {
   }
 
   query(params, callback) {
-    const _params = params || {};
+    let _params = params || {};
```

Comment:
Any particular reason this was changed from a `const` to a `let`? It doesn't affect the correctness of the pull request in any way, but since `_params` doesn't change after it is defined, it's nice to express/guarantee that with a `const`.

Reply:
Thanks for catching this Glenn.  This was a victim of some git rebasing.  It _should_ be a `const` and I will fix.